json-rpc/docs/type_transaction.md

```diff
@@ -115,6 +115,7 @@ The transaction script and arguments of the script call.
 | type           | string       | Name of the script code, see [transaction script doc](../../language/diem-framework/script_documentation/script_documentation.md) for all available script names. If a script function, this field is set to the string `"script_function"`. |
 | code           | string       | Hex-encoded compiled move script bytes        |
 | arguments      | List<string> | List of string value of the script arguments. Contains type information. |
+| arguments_bcs  | List<string> | List of hex-encoded string of BCS bytes representing script function arguments. Does not contain type information. Note that only script function will have `arguments_bcs` value, and the `arguments` field will not be rendered for script function transactions. |
```

Comment:
My bad to not give full guide.
you see the `arguments` line above, it has description as well, you should put the arguments related to there like: This field will not be rendered if `type` is `script_function`.

keep "Note that only script function will have `arguments_bcs` value.|"

Reply:
Sure, sounds fair to me. I'll make the update.

Reply:
Updated accordingly, improved the wording a bit as well.